lms/djangoapps/onboarding_survey/views.py

```diff
@@ -242,3 +268,47 @@ def get_languages(request):
     mime_type = 'application/json'
 
     return HttpResponse(data, mime_type)
+
+
+def update_account_settings(request):
+    """
+    View to handle update of registration extra fields
+    """
+    if request.method == 'POST':
+        user_extended_profile = request.user.extended_profile
```

Comment:
pull this statement out of IF. it is being used in both cases.
```
user_extended_profile = request.user.extended_profile
```

Reply:
Done